Change reference to neo4j database to be the default db for the user (#42)
neo4j/docs-ops-manager: modules/ROOT/pages/installation/server.adoc

Make sure to set a secure password for the built-in admin (`neo4j`) and/or create a specific user for the NOM Server.
Be aware that that user needs full admin access.
The login to the NOM Server is used in the next step.

[IMPORTANT]
====
If the DBMS is not newly created, ensure the following:

** That the default database for the user exists.
This is done via the `SHOW DATABASES` command.
** That the default database for the user is empty.
This is done via `MATCH (n) RETURN count(n)` which should return 0.
** That the default database for the user doesn't contain any indexes or constraints.
This is done via `SHOW INDEXES` and `SHOW CONSTRAINTS`.

> Default database for a user is either the home database for that user if it is set or the default database for the DBMS. 

Reviewer: ```suggestion
The default database for a user is either the home database for that user if it is set or the default database for the DBMS. 
```

> The default database for the DBMS will be 'neo4j' if not explicitly configured.

Reviewer: ```suggestion
The default database for the DBMS is 'neo4j' if not explicitly configured.
```
